.github/workflows/Dockerfile.dev.yml

```diff
@@ -134,17 +136,22 @@ jobs:
     needs: build
     strategy:
       matrix:
+        # TODO when futurenet is up AND master branch of horizon has latest P20 xdr merged in 
+        # network: [testnet, pubnet, standalone, futurenet]
+        # options: ["", "--enable-horizon-captive-core", "--enable-horizon-captive-core --enable-soroban-rpc"]
         network: [testnet, pubnet, standalone]
-        options: ["", "--enable-horizon-captive-core", "--enable-horizon-captive-core --enable-soroban-rpc"]
+        options: ["", "--enable-horizon-captive-core"]
         exclude:
           - network: testnet
             options: "--enable-horizon-captive-core"
           - network: testnet
-            options: "--enable-horizon-captive-core --enable-soroban-rpc"
+            options: "--enable-horizon-captive-core --enable-soroban-rpc" 
           - network: pubnet
             options: "--enable-horizon-captive-core"
           - network: pubnet
             options: "--enable-horizon-captive-core --enable-soroban-rpc"
+          - network: futurenet
+            options: "--enable-horizon-captive-core"
```

Comment:
This exclusion looks unnecessary as futurenet is not included in the list of networks on line 142.

Reply:
I [responded on prior question with reason for commenting out exclusions and options](https://github.com/stellar/quickstart/pull/373/files#r988176426)